### PR TITLE
Add db seeds script

### PR DESCRIPTION
**Pull request by Gabrielpanga**
Add db seeds script

### CODE REVIEW

File: scripts/db/seed.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    }
+    console.log(`Run ${i} seed(s) successfully!`);
+  } catch (err) {
+    console.error(err);
+  }
+
+  process.exit();
+}
+
+if (require.main === module) {
```
Comment (kaiomagalhaes) [inline scripts/db/seed.ts]: why is it necessary?

File: scripts/db/seed.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+import Seeds from '@db/seeds/index';
+import { loadEnvVars } from '@config/initializers/envVars';
+
+async function seed() {
+  console.log('Running seeds');
+
+  try {
+    let i = 0;
+    for (; i < Seeds.length; i++) {
```
Comment (kaiomagalhaes) [inline scripts/db/seed.ts]: could we use `Seeds.forEach` here?

Comment (kaiomagalhaes) [reply scripts/db/seed.ts]: that's interesting, sounds good

File: scripts/db/seed.ts (hunk 1 above)
Comment (kaiomagalhaes) [reply scripts/db/seed.ts]: We should not be worried about using it inside the tests, for those we should have a specific fixtures system

File: src/db/seeds/index.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import UserSeed from './users.seed';
```
Comment (kaiomagalhaes) [inline src/db/seeds/index.ts]: Can you please test if 

`export UsersSeed from './users.seed`

has the same output? this way we wouldn't need the export default

Comment (Gabrielpanga) [reply src/db/seeds/index.ts]: We can't, to do that we have to:
export { default as UserSeeds } from './users.seed';

```
const seedList = Object.keys(Seeds);
    for (; i < seedList.length; i++) {
      await Seeds[seedList[i]]();
    }
```

Im not sure if its cleaner.

**Pull request merged**